src/server/metrics/base.metrics.ts

```diff
@@ -27,7 +27,7 @@ export function createMetricProcessor({
     async update() {
       if (!clickhouse) return;
       const [lastUpdate, setLastUpdate] = await getJobDate(`metric:${name.toLowerCase()}`);
-      const ctx = { db: dbWrite, ch: clickhouse, lastUpdate };
+      const ctx = { db: dbRead, dbWrite, ch: clickhouse, lastUpdate };
```

Comment:
By doing this you'll globally break **all** other metrics as they rely on `db` being `dbWrite` 😬 this can be done, but you must apply this fix to everything.

Reply:
Good point. I was trying to avoid the larger/noisier refactor to `dbRead`, to keep this example minimal, but that will need to be updated for this to work.